### PR TITLE
Fixes /v1/ version incompatibilities

### LINKED ISSUE #40: Breaking change in URL handling
Previously POST /v1/validate/ would work, now it returns 404. That's a bug that should be fixed quickly.

### PR DESCRIPTION
**Pull request by gidsi**
Fixes #40 #41 #42

### CODE REVIEW

File: v1/validator.go [hunk 1]
```diff
@@ -29,14 +29,24 @@ func GetSubMux() *goji.Mux {
 	v1.HandleFunc(pat.Get("/"), info)
 	v1.HandleFunc(pat.Post("/validate"), validate)
 
+	v1.HandleFunc(pat.Get("/validate"), func(writer http.ResponseWriter, _ *http.Request) {
+		writer.WriteHeader(405)
+	})
+	v1.HandleFunc(pat.Get("/validate/"), forwardToValidate)
+	v1.HandleFunc(pat.Post("/validate/"), forwardToValidate)
```
Comment (dbrgn) [inline v1/validator.go]: Shouldn't we offer the main logic at `/validate/` and redirect `/validate` to `/validate/` in order to preserve the previous functionality? It *shouldn't* break robust client code, but many HTTP libraries don't automatically follow redirects.

File: v1/validator.go [hunk 2]
```diff
@@ -29,14 +29,24 @@ func GetSubMux() *goji.Mux {
...
+	http.Redirect(writer, request, "/v1/validate", 302)
+}
+
 func info(writer http.ResponseWriter, request *http.Request) {
 	serverInfo := serverInfo{
 		Description: "Space API Validator API",
-		Usage:       "Send a POST request in JSON format to /v1/validate/. See https://github.com/SpaceApi/validator for more information.",
-		Version:     "1.0.0",
+		Usage:       "Send a POST request in JSON format to /v1/validate. See https://github.com/SpaceApi/validator for more information.",
+		Version:     "2.0.0",
```
Comment (dbrgn) [inline v1/validator.go]: Hm, I was thinking of something like `1.1.0`. It's a bit weird to say that this is version 2.0.0 of the v1 API.

**Review by dbrgn — CHANGES_REQUESTED** [review]
Thanks!

File: v1/validator.go (hunk 2 above)
Comment (gidsi) [reply v1/validator.go]: done

File: v1/validator.go (hunk 1 above)
Comment (gidsi) [reply v1/validator.go]: done

**Review by dbrgn — APPROVED** [review]
Thanks!

**Pull request merged**